deltacat/catalog/model/catalog.py

```diff
@@ -202,6 +203,10 @@ def init(
         logger.warning("DeltaCAT already initialized.")
         return None
 
+    # If no catalogs provided but a config_path exists, create a Catalog
+    if not catalogs and config_path is not None:
```

Comment:
What do we do if a user specifies catalogs AND a config path? I think the easiest initial answer is to just raise a `ValueError`.